VENTokenField/VENTokenField.m

```diff
@@ -44,6 +44,7 @@ @interface VENTokenField () <VENBackspaceTextFieldDelegate>
 @property (strong, nonatomic) VENBackspaceTextField *inputTextField;
 @property (strong, nonatomic) UIColor *colorScheme;
 @property (strong, nonatomic) UILabel *collapsedLabel;
+@property (nonatomic) BOOL hasFocus;
```

Comment:
A comment to the effect of this might help.
Tracks when either of the token fields start editing and stays true if responder status swaps between them.